libs/shared/ui/src/lib/components/buttons/button-icon-action/button-icon-action-element/button-icon-action-element.tsx

```diff
@@ -24,6 +27,20 @@ export function ButtonIconActionElement(props: ButtonIconActionElementProps) {
 
   const [open, setOpen] = useState(false)
 
+  const hasTooltip = (content: React.ReactNode, withRightBorder = false) => {
```

Comment:
The naming of this is a little bit weird. It looks like it's just a boolean check and at the end it prints some dom. 
Could not it be `tooltipWrapper` or something like this?